### ISSUE TITLE
ROL has a problem with Debugger

### ISSUE THREAD

**Issue by Squall-FF8** [t-edson/P65Pas#25]
In this simple code:
```
program NewProgram;
{$ORG $1000}
begin
  asm
    SEC 
    LDA #$FF
    ROL
  end 
end.
```
when I come on ROL in the Debbuger (StepIn) I get error window with Abort/Ignore option.

**Comment by t-edson** [comment]
The issue is fixed in 0.7.8. Please update.
The debugger is the last part implemented so probably not well tested.

**Comment by Squall-FF8** [comment]
It is VERY helpful feature! I write some of my library procedure in asm only and this helps me to test and check results without a need of specific 6502 computer/emulator.
I know there are many resources on the net, but as a part of the Pas65 it really feels as contemporary IDE, excellent work!
